sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -362,6 +361,7 @@ def run_pipeline(self, pipeline):
     result = DataflowPipelineResult(
         self.dataflow_client.create_job(self.job), self)
 
+    from apache_beam.runners.dataflow.dataflow_metrics import DataflowMetrics
```

Comment:
Could you add a comment about imports that are not on top, explaining the reasoning (a JIRA issue, a pylint comment). Anything that would help us to fix the issue in the future will be useful.